lib/debezium/types.go

```diff
@@ -95,57 +95,19 @@ func toInt64(value any) (int64, error) {
 	return 0, fmt.Errorf("failed to cast value '%v' with type '%T' to int64", value, value)
 }
 
-func (f Field) shouldCastToInt64() bool {
-	if f.IsInteger() {
```

Comment:
`IsInteger` only returns true when `Field.DebeziumType` does not match one of our handled types AND `Field.Type` equaled `Int16`, `Int32`, or `Int64`.